### DISCUSSION TITLE
an element with the same key has already been added

### DISCUSSION THREAD

**Discussion by nuclead-gnome** [wiresock/WireSockUI#56] [General]
Before reinstalling Windows, the program did not issue any errors or warnings, now when adding the same configurations it writes - An element with the same key has already been added.
How can I fix this?
![Безымянный](https://github.com/wiresock/WireSockUI/assets/159523384/7f432991-6f60-49fc-9671-699dc0646a72)

**Comment by wiresock** [comment]
This feature is supported in WireSock VPN Client v1.4.5 and WireSockUI v0.2.6.
